protodf/__init__.py

```diff
@@ -71,5 +75,13 @@ def __to_row_data(field_descriptor, data):
     if field_descriptor.message_type is not None:
         return message_to_row(field_descriptor.message_type, data)
     if field_descriptor.type == field_descriptor.TYPE_ENUM:
-        return field_descriptor.enum_type.values[data].name
+        return __enum_to_row_data(field_descriptor, data)
     return data
+
+
+def __enum_to_row_data(field_descriptor, data):
+    try:
+        enum_value = field_descriptor.enum_type.values_by_number[data]
+        return {"name": enum_value.name, "number": enum_value.number}
```

Comment:
Is this indeed how it is represented in a Row? You said you had some tests you wanted to add? That would be great to see this actually works.

Reply:
I've added unit tests.
Btw, I realized that default values are represented as `None`. I wonder if that can cause unexpected errors or behaviour in my PySpark job...

Reply:
I'm not a Python developer. Feedback on implementation is welcome :-)
Btw, what source/doc reference did you use when implementing `protodf`?

Reply:
Changed PR to draft. I'm not satisfied with my work yet.